cloud/google/clusteractuator.go

```diff
@@ -186,6 +209,21 @@ func (gce *GCEClusterClient) deleteFirewallRule(cluster *clusterv1.Cluster, rule
 	return gce.computeService.WaitForOperation(clusterConfig.Project, op)
 }
 
+func (gce *GCEClusterClient) handleClusterError(cluster *clusterv1.Cluster, err *apierrors.ClusterError, eventAction string) error {
+	if gce.v1Alpha1Client != nil {
+		reason := err.Reason
+		message := err.Message
+		cluster.Status.ErrorReason = reason
```

Comment:
is there a reason not to set the ErrorReason directly with err.Reason?

Reply:
I don't think so, I updated this to use err.Reason and err.Message directly